ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -319,6 +319,7 @@ const IgvPanel = React.memo(({ variant, igvSampleIndividuals, individualsByGuid,
     variant.chrom,
     (variant.genomeVersion !== project.genomeVersion && variant.liftedOverPos) ? variant.liftedOverPos : variant.pos,
     100,
+    variant.end && variant.end - variant.pos,
```

Comment:
It seems that we need to increase the range and not just add an end offset.

Reply:
no, we want the start position of the locus to be relative to the start of the SV and the end position to be relative to the end of the SV, and if you pass the endOffset to `getLocus` that is exactly what happens